test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua

```diff
@@ -0,0 +1,111 @@
+   {rpc = "DeleteInteractionChoiceSet", params= {interactionChoiceSetID = 1}},
+   {rpc = "Alert", params = {}},
+   {rpc = "Show", params = {mediaClock = "00:00:01", mainField1 = "Show1", softButtons = {{ text = "", systemAction = "DEFAULT_ACTION", type = "BOTH", isHighlighted = true, image = {imageType = "DYNAMIC", value = "icon.png"},  softButtonID = 3 }}}},
+   {rpc = "EndAudioPassThru", params = {}},
+   {rpc = "SubscribeVehicleData", params = {gps = true}},
+   {rpc = "UnsubscribeVehicleData", params = {gps = true}},
+   {rpc = "GetVehicleData", params = {speed = true}},
+   {rpc = "ScrollableMessage", params = {scrollableMessageBody = commonFunctions:createString(500), softButtons = {{softButtonID = 1, text = commonFunctions:createString(500),type = "TEXT", isHighlighted = false, systemAction = "DEFAULT_ACTION"}}}},
+   {rpc = "Slider", params = {numTicks = 7, position = 6, sliderHeader ="sliderHeader"}},
+   {rpc = "ShowConstantTBT", params = {}},
+   {rpc = "AlertManeuver", params = {ttsChunks = {{text ="FirstAlert", type ="TEXT",}, {text ="SecondAlert", type ="TEXT",},}}},
+   {rpc = "ChangeRegistration", params = {language ="EN-US", hmiDisplayLanguage ="EN-US", appName ="SyncProxyTester", ttsName = {{text ="SyncProxyTester",type ="TEXT",},}, ngnMediaScreenAppName ="SPT",  vrSynonyms = { "VRSyncProxyTester", }, }},
+   {rpc = "SetAppIcon", params = {syncFileName = "icon.png"}},
+   {rpc = "SetDisplayLayout", params = {displayLayout = "ONSCREEN_PRESETS"}},
+   {rpc = "SendLocation", params = {address = {countryName = "countryName"}}},
+ --{rpc = "DialNumber", params = {{number = "#3804567654*"}}},
+ --{rpc = "UpdateTurnList", params = {}}
+ --{rpc = "Speak", params = {}}, 
+ --{rpc = "SetMediaClockTimer", params = {}}
+ --{rpc = "PerformAudioPassThru", params = {}}
+ --{rpc = "PerformInteraction", params = {}},
+ --{rpc = "SubscribeButton", params = {}}, 
+ --{rpc = "UnsubscribeButton", params = {}},    
+-- {rpc = "GenericResponse", params =   {}}                   
+-- {rpc = "SystemRequest", params = {}},
+ --{rpc = "GetWayPoints", params = {}},
+-- {rpc = "SubscribeWayPoints", params = {}},
+-- {rpc = "UnsubscribeWayPoints", params = {}},
+-- ATF issue - "mandatory parameter spaceAvailable not present"
+-- {rpc = "ListFiles", params = {}},
+ --{rpc = "PutFile", params = {syncFileName ="icon.png", fileType ="GRAPHIC_PNG", persistentFile =false, systemFile = false, offset =0, length =11600}},
+ --{rpc = "DeleteFile", params = {syncFileName = "test.png"}},
+}
+
+--[[ General Precondition before ATF start ]]
+--Copy attached json file in SDL_bin folder.
+--Json contains application 123abc with NULL policies
+commonSteps:DeleteLogsFileAndPolicyTable()
+--[[ Preconditions ]]
+Test = require('connecttest')
+function Test:OpenNewSession()
+commonFunctions:userPrint(33, "================= Precondition ==================")
```

Comment:
Please remove the print should be called as indicated above. It is put immediately after name of section.